CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* Fix focus state on step by step nav when header element is hovered [PR #1119](https://github.com/alphagov/govuk_publishing_components/pull/1119)
```

Comment:
Nitpick: can this match the PR name?

Reply:
Sure, but I'll do it the other way around - updated the PR name as it's more accurately described in the changelog.